api/tests/app/letters/test_letter_schemas.py

```diff
@@ -7,6 +7,7 @@
 from app.letters.letter_schemas import letter_job_ids
 from app.schema_validation import validate
 
+pytest.skip("we no longer support sending letters", allow_module_level=True)
```

Comment:
Do you think we'll be supporting letters in the future? Wonder if we should just remove these files.

Reply:
This implies we previously supported them. Might we worth wording it "we do not support sending letters" just so any readers are not confused.